inst/rmarkdown/homepage/index.Rmd

```diff
@@ -0,0 +1,363 @@
+The latter is more of a convenience to aid quick development and employment of protocols within a project context.
+Within the scope of a project, it might sometimes be necessary to re-use a `sfp` type protocol but with slightly different settings which are specific to the project.
+This can be done by including a published `sfp` protocol as an `spp` subprotocol where the deviations from the default settings are given.
+
+
+### Protocol identification
+
+The previously explained typology of protocols is further detailed in a so-called protocol-code.
+A **protocol-code** consists of a prefix referring to the *protocol-type* (three characters), a *protocol-number* (three digits) and a *language tag* (two characters), all separated by a hyphen.
+
+| type       | theme      | theme_number | protocol-code |
+|:-----------|:-----------|:-------------|:--------------|
+| field      | generic    | 0            | sfp-0##-en    |
+| field      | water      | 1            | sfp-1##-en    |
+| field      | air        | 2            | sfp-2##-en    |
+| field      | soil       | 3            | sfp-3##-en    |
+| field      | vegetation | 4            | sfp-4##-en    |
+| field      | species    | 5            | sfp-5##-en    |
+| instrument |            |              | sip-###-en    |
+| operating  |            |              | sop-###-en    |
+| analysis   |            |              | sap-###-en    |
+| project    |            |              | spp-###-en    |
+
+The three digits of the protocol-code will be referred to as the *protocol-number*.
+In case of thematic protocols, the first digit of the protocol-number corresponds with the theme-number.
+The `##` indicates an incremental number.
+For instance, the first field protocol for "theme water" will have protocol code `sfp-101-en`.
+The `s*p-###` part of the protocol code can be thought of as a code that corresponds one-to-one with the title of the protocol (when ignoring language).
+
+The final two characters identify the language the protocol is written in.
+This can be either Dutch (`nl`) or English (`en`).
+
+Each protocol thus has a unique protocol-code, but there can be multiple versions of a protocol.
+To discriminate between different versions of protocol, we need a version number.
+The **version number** that we use is of the form `YYYY.NN`.
+`YYYY` indicates the year in which the protocol was released.
+The `NN` is a number that indicates the order of release within that year (starting with 01).
+The version number will therefore uniquely identify a specific version of a protocol even without the need to refer to the protocol-code.
+Here's a fictive example:
+
+- version `2022.01` (identifies a first version of `sfp-401-en`)
+- version `2022.02` (identifies a first version of `sfp-001-en`)
+- version `2022.03` (identifies a second version of `sfp-401-en`)
+
+The NEWS page lists the version numbers and identifies which protocols they belong to.
```

Comment:
Link to NEWS page?

Reply:
Done in [13b6571](https://github.com/inbo/protocolhelper/pull/97/commits/13b65714278609708268f5625505a54d5939d37f)